common/app/templates/headerInlineJS/bootCurl.scala.js

```diff
@@ -42,6 +42,7 @@ var curl = {
 @JavaScript(Static.js.curl)
 
 require([
+    'common/utils/config',
```

Comment:
That makes sense to me, though I might be tempted to add a comment at this point, as it might be easy to overlook why this is here.